examples/smux.rs

```diff
@@ -213,11 +224,29 @@ impl PtyPane {
                     writer.write_all(&bytes).unwrap();
                     writer.flush().unwrap();
                 }
-                drop(pty_pair.master);
```

Comment:
Please also remove the corresponding comment.

Reply:
I removed the unnecessary explicit scope and updated the comment